templates/aws/workspaces.html

```diff
@@ -17,34 +17,33 @@
 {% block content %}
   <section class="p-section--hero">
     <div class="row--50-50">
-      <div class="col p-strip is-shallow">
+      <div class="col">
         <h1>Ubuntu WorkSpaces</h1>
-        <p class="p-heading--5 u-no-margin--bottom">
+        <p class="p-heading--5">
           Virtual desktops, powered by AWS.
           <br />
           Develop remotely, securely and cost-effectively.
         </p>
-        <div class="p-strip is-shallow">
-          OpenSearch is an open source search and analytics suite that developers use to build solutions for search, data observability, data ingestion and more.
-          It is designed for scalability, offering powerful full-text search capabilities and supporting various data types, including structured and unstructured data. OpenSearch has rapidly developed into a standalone platform with unique features and capabilities.
-        </div>
+        <p>Ubuntu Desktop on Amazon WorkSpaces offers developers an optimised Ubuntu Desktop environment to rapidly build, test and deploy code &ndash; spinning up and tearing down instances as required.</p>
+        <p>IT organisations benefit from rapid deployment, configuration and scalability for their managed Ubuntu Desktops whilst their applications and data remain secured in the AWS cloud, even when their developers are working remotely.</p>
       </div>
-      <div class="col p-strip is-shallow">
-        <div class="p-image-container--16-9 is-highlighted u-hide--medium u-hide--small">
-          {{ image(url="https://assets.ubuntu.com/v1/6143478c-correct-bg-aws.png",
-                    alt="AWS Workspaces",
-                    width="300",
-                    height="200",
-                    hi_def=True,
-                    loading="lazy",
-                    attrs={"class": "p-image-container__image"}) | safe
-          }}     
+      <div class="col">
+        <div class="p-section--shallow">
+          <div class="p-image-container--16-9 is-highlighted u-hide--medium u-hide--small">
+            {{ image(url="https://assets.ubuntu.com/v1/6143478c-correct-bg-aws.png",
+                      alt="AWS Workspaces",
```

Comment:
```suggestion
                      alt="",
```
Can probably remove this, it isn't adding much